### PR TITLE
fix deprecated function name and add scalarbar3d

### PR DESCRIPTION
**Pull request by j042**

### CODE REVIEW

**Comment by clemens-fricke** [comment]
Can you modify an example to include the 3D scalar bar or have a test for this snippet's functionality?

**Comment by j042** [comment]
> Can you modify an example to include the 3D scalar bar or have a test for this snippet's functionality?

done

**Comment by clemens-fricke** [comment]
I think the example is a little suboptimal. At the default position, you can not see the scalarbar in this plot. 
image
But it is there if you turn the object a little.
image

**Comment by j042** [comment]
@clemens-fricke what about this example?

File: gustaf/show.py [hunk 1]
```diff
@@ -317,11 +317,17 @@ def _vedo_showable(obj, as_dict=False, **kwargs):
...
             sb_kwargs = dict() if isinstance(sb_kwargs, bool) else sb_kwargs
-            vedo_obj.addScalarBar(**sb_kwargs)
+            vedo_obj.add_scalarbar(**sb_kwargs)
+
+        sb3d_kwargs = obj.show_options.get("scalarbar3d", None)
+        if sb3d_kwargs is not None and sb3d_kwargs is not False:
+            sb3d_kwargs = (
+                dict() if isinstance(sb3d_kwargs, bool) else sb3d_kwargs
+            )
+            vedo_obj.add_scalarbar3d(**sb3d_kwargs)
```
Comment (clemens-fricke) [inline gustaf/show.py:330]: Do you know what this/vedo will do of both scalarbar and 3d scalarbar are added?

**Review by clemens-fricke — APPROVED** [review]
Seems to be a straightforward feature addition. Thank you, also for the added example.

File: gustaf/show.py (hunk 1 above)
Comment (j042) [reply gustaf/show.py:330]: it will show you both

**Pull request merged**